posts/templates/posts/_post_create.html

```diff
@@ -7,10 +7,20 @@
             class="form-control floating-label {% if 'content' in form.errors %} has-error{% endif %}"
             rows="3"
             v-model="content"
-            data-hint="(% 256 - content.length%)"
+            data-hint="(% 256 - content.length %)"
             placeholder="Write something!">{{ post.content | escape }}</textarea>
     <button class="btn btn-lg btn-raised btn-primary pull-right" type="submit">
         {% if post %}Save{% else %}Write{% endif %}
     </button>
+    {% if user.circles %}
+        <div class="form-group col-md-5 col-xs-6 col-sm-6 col-lg-4 pull-right">
+            <select class="form-control" name="circle">
+                <option disabled selected="selected">Select a circle</option>
```

Comment:
Either we remove this disabled option and the users has to always pick a cirlce, or we need to catch the case that no circle is selected in python. Right now there's an error.
![download](https://cloud.githubusercontent.com/assets/872251/8360091/bb0d6222-1b6a-11e5-9b82-ccdf4e7d7454.png)